packages/core/src/build/service.ts

```diff
@@ -48,15 +54,24 @@ export type Build = {
   networks: Network[];
   // Schema
   schema: Schema;
-  graphqlSchema: GraphQLSchema;
+  graphQLSchema: GraphQLSchema;
   // Indexing functions
   indexingFunctions: IndexingFunctions;
 };
 
+export type BuildServer = {
+  app: Hono;
+  routes: PonderRoutes;
+};
+
 export type BuildResult =
   | { status: "success"; build: Build }
   | { status: "error"; error: Error };
 
+export type BuildResultServer =
+  | { status: "success"; build?: BuildServer }
+  | { status: "error"; error: Error };
```

Comment:
Similar to comment above - I think this would make more sense as `IndexingBuildResult` and `ApiBuildResult`.